lib/bcdice/game_system/AlchemiaStruggle.rb

```diff
@@ -0,0 +1,353 @@
+# frozen_string_literal: true
+
+require "bcdice"
+
+module BCDice
+  module GameSystem
+    class AlchemiaStruggle < Base
+      ID = "AlchemiaStruggle"
+      NAME = "アルケミア・ストラグル"
+      SORT_KEY = "あるけみあすとらくる"
+
+      HELP_MESSAGE = <<~MESSAGETEXT
+        ■ ダイスロール（ XAS ）
+          ＸＤをロールします。
+          例） 5AS
+        
+        ■ ダイスロール＆最大になるようにピック（ XASY ）
+          ＸＤをロールし、そこから最大になるようにＹ個をピックします。
+          例） 4AS3
```

Comment:
```suggestion
        ■ ダイスロール（ xAS ）
          xDをロールします。
          例） 5AS
        
        ■ ダイスロール＆最大になるようにピック（ xASy ）
          xDをロールし、そこから最大になるようにy個をピックします。
          例） 4AS3
```

数字の入力場所とキーワードが区別しやすいようにしてください